history/characters/340000_night_elf.txt

```diff
@@ -1,16 +1,52 @@
-#Dynasty 128000
+#Dynasty 128000 # Mar'alith
 340000={
-	name=Mar'alith
+	name=Pas'draen
 	dynasty=128000
-	dna="kinkbpfiibj"
+	dna="kpekbpgnfeg"
 	culture=night_elf religion=kaldorei_religion
 	martial=5 diplomacy=6 stewardship=7 intrigue=3 learning=8
 	add_trait=skilled_tactician add_trait=diligent add_trait=brave add_trait=patient add_trait=strategist
 	disallow_random_traits = yes
-	1.9.23={ birth=yes add_trait=creature_night_elf immortal_age=18 }
+	1.12.21={ birth=yes add_trait=creature_night_elf immortal_age=18 }
 	20.1.1={ add_trait=class_warrior_6 }
+	25.1.1={
+			add_spouse=340001
+			effect={
+					add_lover=340001
+			}
+	}
 	1059.11.25={ death=yes }
 }
+340001={
+	name=Natalia
+	female=yes
+	dynasty=128000
```

Comment:
She's from the husband's dynasty, so it looks like incest.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/76784838-c9cf8b00-67cd-11ea-846a-1c747798b1b5.png)

</details>

Reply:
can be resolved by making the commander marry into her dynasty matrilineally due to matriarchal society of the kaldorei

Reply:
He needs a dynasty though. He's a feudal ruler.

Reply:
will just remove dynasty from Natalia instead